nipype/interfaces/ants/segmentation.py

```diff
@@ -31,7 +31,12 @@ class AtroposInputSpec(ANTSCommandInputSpec):
         requires=["number_of_tissue_classes"],
         mandatory=True,
     )
-    prior_probability_images = InputMultiPath(File(exists=True))
+    kmeans_init_centers = traits.List(traits.Either(traits.Int, traits.Float), minlen=1)
+    prior_image = traits.Either(
+        File(exists=True),
+        traits.Str,
+        desc="either a string pattern (e.g., 'prior%%02d.nii') or an existing vector-image file.",
```

Comment:
Is this string being interpolated somewhere?

```suggestion
        desc="either a string pattern (e.g., 'prior%02d.nii') or an existing vector-image file.",
```